src/charts/brush-range-editor.js

```diff
@@ -65,10 +87,32 @@ export default function BrushRangeEditor (_container) {
       cache.inputMin = cache.root.append("div")
         .attr("class", "brush-range-input min")
         .attr("contentEditable", true)
+        .on("focus", function focus () {
+          let text = cache.inputMin.text()
+          const parsed = d3.timeParse("%b %d, %Y")(text)
+          if (parsed instanceof Date) {
+            text = d3.timeFormat("%m-%d-%Y")(parsed)
+          }
+          cache.inputMin.text(text)
```

Comment:
This doesn't need to be set if it was not a date, so it could move into the if block above.